
syntax = "proto3";

package descarteslabs.workflows;

message LogRecord {
  enum Level {
    DEBUG = 0;
    INFO = 1;
    WARNING = 2;
    ERROR = 3;
  }
  Level level = 1;
  string message = 3;
  int64 timestamp = 4;
}
